utils/get_abstracts.py

```diff
@@ -0,0 +1,84 @@
+import synapseclient
+from synapseclient import Table
+import argparse
+import pandas as pd
+import requests
+import json
+
+
+### Login to Synapse ###
+def login():
+
+    syn = synapseclient.Synapse()
+    syn.login()
+
+    return syn
+
+
+def get_args():
+
+    parser = argparse.ArgumentParser(
+        description='Get synapse publications table id')
+    parser.add_argument('table_id',
+                        type=str,
+                        help='Synapse table id to upload manifest to.')
+
+    return parser.parse_args()
+
+
+def get_df(syn, publications_table_id):
+
+    pubs_query = (f"SELECT pubMedId, abstract FROM {publications_table_id}")
+    pubs_df = syn.tableQuery(pubs_query).asDataFrame().fillna("")
+
+    pmid_list = pubs_df['pubMedId'].tolist()
+
+    return pubs_df
+
+
+def get_pmids(pubs_df):
+
+    # Convert column of pubmedIds to a list
+    pmid_list = pubs_df['pubMedId'].tolist()
```

Comment:
Redundant with line 34.

Should `get_df()` be returning `pmid_list`?  If so, then I recommend removing this function.

Reply:
No, I removed the other one because I still need the df from the other function and the list from this function. Thanks for catching!